apps/teca_bayesian_ar_detector_parameters.h

```diff
@@ -0,0 +1,6 @@
+using parameter_t = double;
```

Comment:
This file needs to be rewritten to contain actual parameters from the MCMC estimation - this two-parameter header is just for testing.